lib/components/query.js

```diff
@@ -91,27 +91,27 @@ querie.endpoint = function (nwkAddr, epId, callback) {
     }).nodeify(callback);
 };
 
-querie.deviceWithEndpoints = function (nwkAddr, ieeeAddr, callback) {
+query.deviceWithEndpoints = function (nwkAddr, ieeeAddr, callback) {
     var deferred = Q.defer(),
-        epQueries = [],
+        epquerys = [],
         fullDev;
 
-    querie.device(ieeeAddr, nwkAddr).then(function (devInfo) {
+    query.device(ieeeAddr, nwkAddr).then(function (devInfo) {
         fullDev = devInfo;
 
         _.forEach(fullDev.epList, function (epId) {
-            var epQuery = {func: querie.endpoint, nwkAddr: nwkAddr, epId: epId};
-            epQueries.push(epQuery);
+            var epQuery = {func: query.endpoint, nwkAddr: nwkAddr, epId: epId};
+            epquerys.push(epQuery);
```

Comment:
querys should be queries (https://www.dictionary.com/browse/queries)

Reply:
Totally agreed, will fix.

Reply:
Donezo.